### DISCUSSION TITLE
need help control times for hot water or night setback

### DISCUSSION THREAD

**Discussion by CRU01** [emsesp/EMS-ESP32#1858] [Q&A]
Hey guys,
So far I've only read here and I'm impressed.
Since I haven't found a solution yet, I would like to try something out as a beginner, although I don't know whether my research is enough.
I have no knowledge of heating and/or controls, but I look after our clubhouse 100 km away fromm y home, so I installed the EMS-ESP gateway.
Since 11/23 we use a Bosch Condens 7800 iW; UI800/BC400 and EMS-ESP Gateway (EMS-ESP Version 3.6.5)
Since 04/24 we use a 17 m2 Solar thermal collector with 750 liter buffer storage with the following Bosch controls MM100, MM200, MM200, SM200/MS200, M100/MS100.
After initial problems, the system is currently running, but there are apparently still opportunities for optimization

I'm currently using the EMS-ESP for monitoring and minor adjustments and I'm thrilled with it.

On site you can set the control times for hot water or night setback at the thermal bath.
I didn't discover this in the EMS-ESP, can you also activate this part of the control so that you can change it remotely?

kind regards
Christian

**Comment by proddy** [comment]
Can you show how you set these on the physical device - is via the boiler, thermostat? If we haven't yet implemented it, we would need the data which is difficult since you're not on site to capture the telegrams real-time. An alternative is to use the Scheduler in EMS-ESP to turn things on/off at set periods.

**Comment by CRU01** [comment]
The setting is made on the touchscreen of the hot water heater display in the menu. I can take photos of the page structure.

**Reply by proddy** [reply]
any chance to capture some images?

**Reply by CRU01** [reply]
I can´t update the new version of the latest 3.70 Version, because of this
![grafik](https://github.com/user-attachments/assets/655f92df-57e1-40a1-96df-04101787e774)
